docs/sdk/dev/development-tools/ts-linter.md

```diff
@@ -41,6 +41,12 @@ related:
     link: docs/scos/dev/sdk/development-tools/tooling-config-file.html
 ---
 
+{% info_block warningBox "No longer supported" %}
+
+Since `202304.0` release this tool is no longer supported and was replaced by `ESLint`. If you want to migrate on `ESLint`, see [Migration guide - Switch from TSLint to ESLint](/docs/scos/dev/front-end-development/{{site.version}}/migration-guide-switch-from-tslint-to-eslint.html).
```

Comment:
```suggestion
Since the `202304.0` release, this tool is no longer supported and has been replaced by `ESLint`. Follow the [migration guide](/docs/scos/dev/front-end-development/{{site.version}}/migration-guide-switch-from-tslint-to-eslint.html) to migrate to `ESLint`.
```